pkg/server/container_stats_list_windows.go

```diff
@@ -76,18 +76,20 @@ func (c *criService) getSandboxMetrics(
 			Annotations: meta.Config.GetAnnotations(),
 		},
 	}
-	v, err := typeurl.UnmarshalAny(stats.Data)
-	if err != nil {
-		return nil, err
-	}
-	if s, ok := v.(*runhcsstats.Statistics); ok {
-		cs.Cpu = &runtime.CpuUsage{
-			Timestamp:            stats.Timestamp.UnixNano(),
-			UsageCoreNanoSeconds: &runtime.UInt64Value{s.VM.Processor.TotalRuntimeNS},
+	if stats != nil {
```

Comment:
Upstream wouldn't allow this invariant why are we? I mean what case returns no error but nil stats?

Reply:
Upstream has this exact same check: https://github.com/containerd/cri/blob/master/pkg/server/container_stats_list_unix.go#L59

Due to the way they are processing stats, if the stats call returns an error they will actually end up calling this function with nil. I assume the intent is to still output a stats object even in the case of error.

Reply:
Oh I see. I only checked our fork ok cool. Thanks